internal/telemetry/event.go

```diff
@@ -34,6 +35,18 @@ func TrackDeploymentCreate(id, deploymentID, template string, dryRun bool) error
 	return TrackEvent(deploymentCreateEvent, id, props)
 }
 
+func TrackDeploymentHeartbeat(id, deploymentID, clusterID, template, templateVersion, targetInfrastructure string) error {
+	props := map[string]interface{}{
+		"hmcVersion":           build.Version,
+		"deploymentID":         deploymentID,
+		"clusterID":            clusterID,
+		"targetInfrastructure": targetInfrastructure,
```

Comment:
I think this should be infrastructureProvider in CAPI terms (and we can also add other providers - bootstrap and controlplane just in case) @Schnitzel

Reply:
I like the updated ones, good for me to go!